pyomo/contrib/mindtpy/nlp_solve.py

```diff
@@ -197,19 +243,34 @@ def handle_NLP_subproblem_other_termination(fixed_nlp, termination_condition,
 
 
 def solve_NLP_feas(solve_data, config):
-    """Solves feasibility NLP and copies result to working model
-
-    Returns: Result values and dual values
     """
-    fixed_nlp = solve_data.working_model.clone()
-    add_feas_slacks(fixed_nlp, config)
+    Handles the result of the latest iteration of solving the NLP subproblem given a feasible, non-optimal solution;
+    solves the feasibility NLP and copies result to working model
+    TODO: What does the function do (does it just update the bounds?)
```

Comment:
This function solves a feasibility NLP problem after the fixed_nlp problem is infeasible